src/storage/files.js

```diff
@@ -51,7 +52,7 @@ class FilesStorage {
       filename += `-${('0' + date.getSeconds()).slice(-2)}`
     }
 
-    return filename.replace(/\s+/g, '')
+    return path.join(this.options.filesLocation, filename.replace(/\s+/g, ''));
```

Comment:
this.options.filesLocation is concatened twice (here and line 37)

Reply:
Thanx, fixed 👍